influxdb-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,118 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'influxdb',
+        rules: [
+          {
+            alert: 'InfluxDBWarningTaskSchedulerHighFailureRate',
+            expr: |||
+              100 * rate(task_scheduler_total_execute_failure[5m])/clamp_min(rate(task_scheduler_total_execution_calls[5m]), 1) >= %(alertsWarningTaskSchedulerHighFailureRate)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'Automated data processing tasks are failing at a high rate.',
+              description:
+                (
+                  'Task scheduler task executions for instance {{$labels.instance}} on cluster {{$labels.influxdb_cluster}} are failing at a rate of {{ printf "%%.0f" $value }} percent, ' +
+                  'which is above the threshold of %(alertsWarningTaskSchedulerHighFailureRate)s percent.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'InfluxDBCriticalTaskSchedulerHighFailureRate',
+            expr: |||
+              100 * rate(task_scheduler_total_execute_failure[5m])/clamp_min(rate(task_scheduler_total_execution_calls[5m]), 1) >= %(alertsCriticalTaskSchedulerHighFailureRate)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'Automated data processing tasks are failing at a high rate.',
```

Comment:
if you really want you can keep it the same since the title differentiates the alert, but it would be better if you can say something like 'Automated data processing tasks are failing at a critically high rate.',